### PR TITLE
More and cleaner project pages

### PR DESCRIPTION
**Pull request by coreycaitlin**
See issue(s) #2202 and #2275 

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/xtra-project-pages)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/18f.gsa.gov/xtra-project-pages/)

Changes proposed in this pull request:
- add the C2 intro
- tidy up markdown and frontmatter for consistency
- add `quote` and `learn_more` elements to frontmatter
- make `project_url` markdown-friendly

/cc @austinhernandez @gemfarmer

### CODE REVIEW

**Comment by coreycaitlin** [comment]
handing off to @gemfarmer to...

- fix blockquote styling (why so much left padding?!)
- add some simple list styling for ul (relevant for client lists and resources lists)
- fix the "all posts" link by adding `/tag` in the middle of the url, which I forgot

**Comment by coreycaitlin** [comment]
Oh! and we should change the path from `/project/` to `/what-we-deliver/`

File: _sass/_components/section-info.scss [hunk 1]
```diff
@@ -11,14 +11,41 @@
...
+
+    ul:not(.usa-unstyled-list) {
+      padding-left: $paragraph-margins;
+      margin-top: $spacer-margins;
+
+      li {
+        line-height: $heading-line-height;
+        list-style-type: disc;
+
+        &:last-of-type {
```
Comment (gemfarmer) [inline _sass/_components/section-info.scss]: oi oi oi

**Comment by gemfarmer** [comment]
The hound errors are just repeats of before. We could fix them, but it would require a bit of restructuring.

**Comment by gemfarmer** [comment]
@coreycaitlin when these changes pass, it should be good to go!

**Comment by gemfarmer** [comment]
This build failure can safely be ignored.

**Pull request merged**